### PR TITLE
updated descriptions

### PR DESCRIPTION
**Pull request by McP4nth3r**
added SignalStrength available values
added Volume available values

### CODE REVIEW

File: commands.md [hunk 1]
```diff
@@ -145,8 +145,8 @@ Parameter object:
 Property | Type | Description
 ------------ | ------------- | -------------
 Name | `string` | TeamSpeak name of the player
-SignalStrength | `int` | Determines the level of voice distortion (0 = Good qulity)
-Volume | `float?` | Overrides the volume
+SignalStrength | `int` | Determines the level of voice distortion (0 = Good qulity, 18 = worst quality)
```
Comment (BlackFlash5) [inline commands.md:148]: The range is 0-20
```suggestion
SignalStrength | `int` | Determines the level of voice distortion (0 = Good qulity, 20 = worst quality)
```

File: commands.md [hunk 2]
```diff
@@ -145,8 +145,8 @@ Parameter object:
...
+Volume | `float?` | Overrides the volume `available (0 - 20)`
```
Comment (BlackFlash5) [inline commands.md:149]: The range is 0-1.6
```suggestion
Volume | `float?` | Overrides the volume in percent (0f - 1.6f / 0-160%)
```

File: commands.md [hunk 3]
```diff
@@ -197,7 +197,7 @@ Name | `string` | TeamSpeak name of the player
 SenderRadioType | `RadioType` | Radio type of the sending player
 OwnRadioType | `RadioType` | The local players radio type
 PlayMicClick | `bool` | If `true` Salty Chat will automatically play the sound "onMicClick" or "offMicClick" of the specified sound pack, if the player is in range
-Volume | `float?` | Overrides the volume
+Volume | `float?` | Overrides the volume `available (0 - 20)`
```
Comment (BlackFlash5) [inline commands.md:200]: The range is 0-1.6
```suggestion
Volume | `float?` | Overrides the volume in percent (0f - 1.6f / 0-160%)
```

File: commands.md [hunk 4]
```diff
@@ -297,7 +297,7 @@ Property | Type | Description
 ------------ | ------------- | -------------
 Name | `string` | TeamSpeak name of the player
 Range | `float` | Range where the player can be heard through the megaphone
-Volume | `float?` | Overrides the volume
+Volume | `float?` | Overrides the volume `available (0 - 20)`
```
Comment (BlackFlash5) [inline commands.md:300]: The range is 0-1.6
```suggestion
Volume | `float?` | Overrides the volume in percent (0f - 1.6f / 0-160%)
```

**Comment by McP4nth3r** [comment]
Ok thanks for the answer you‘ve write 0-20 but I think you mean SignalStrength

**Comment by BlackFlash5** [comment]
Stale PR

**Pull request closed without merging**